dist/templates/layout.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```

Comment:
Exactly, they were built by some task - we should only track `src` that is the input of task and task body itself. Result of task shouln't be in repository as it is determined by source and gulp's task.